### ISSUE TITLE
Compiler hangs on open.mp libs 

### ISSUE THREAD

**Issue by PerseweratywneImponderabilia** [ZantetsukenGT/samp-dyn-dialog-pages#1]
It appears the compiler hangs while using the open.mp libs. It only reports this warning:
```
/home/mysy/Projects/samp/samp-dyn-dialog-pages/dyn-dialog-pages.inc:248 (warning) tag mismatch: expected tag "t_DIALOG_STYLE", but found none ("_")
/home/mysy/Projects/samp/samp-dyn-dialog-pages/ddp-examples.pwn:56 (warning) symbol is never used: "ShowPlayerDialog@01"
Build failed with 2 problems
```

I've tried to change the used native to open.mp's variation, sadly I got an error instead. I wouldn't be surprised if I somehow adjusted the hook incorrectly but I'm clueless here.
```
/home/mysy/Projects/samp/samp-dyn-dialog-pages/dyn-dialog-pages.inc:248 (error) undefined symbol "_@yHShowPlayerDialog@01"; did you mean "@yH_ShowPlayerDialog@01"?
/home/mysy/Projects/samp/samp-dyn-dialog-pages/ddp-examples.pwn:56 (warning) symbol is never used: "ShowPlayerDialog@01"
Build failed with 2 problems
```

The new `ShowPlayerDialog`:
```
native bool:ShowPlayerDialog(playerid, dialogid, DIALOG_STYLE:style, const title[], const body[], const button1[], const button2[], OPEN_MP_TAGS:...);
```

Compiler used: 3.10.11

**Comment by ZantetsukenGT** [comment]
I'll take a look at this today, when I made this library i used the usual compiler and the usual samp libraries.
With omp some signatures are different, so I need to change them

**Comment by ZantetsukenGT** [comment]
Fixed by c065a0a541694a4742aa896e20499cae442d8da2
For some odd reason 3.10.11 crashes whenever you use `#warning`. So originally the library was trying to include `a_samp`, and the new omp libs issue a warning when you do so

**Comment by ZantetsukenGT** [comment]
However, #2
